framework/Models/PostProcessors/TopologicalDecomposition.py

```diff
@@ -377,15 +377,15 @@ class cls.
       inputSpecification.addSub(InputData.parameterInputFactory("interactive"))
       return inputSpecification
 
-    def __init__(self, runInfoDict):
+    def __init__(self):
       """
        Constructor
-       @ In, messageHandler, message handler object
+       @ In, None
        @ Out, None
       """
-
-      TopologicalDecomposition.__init__(self, runInfoDict)
-      qtc.QObject.__init__(self)
+      super().__init__()
+      # TopologicalDecomposition.__init__(self)
+      # qtc.QObject.__init__(self)
```

Comment:
remove commented lines

Reply:
I actually left these as a guide in the event `super()` resolution breaks the performance. If you believe there's small chance of that happening, though, I can remove them.